scripts/create_kind_cluster.sh

```diff
@@ -25,6 +25,34 @@ containerdConfigPatches:
 - |-
   [plugins."io.containerd.grpc.v1.cri".registry.mirrors."localhost:${reg_port}"]
     endpoint = ["http://${reg_name}:${reg_port}"]
+nodes:
```

Comment:
each user has their application with a new port.... because with node port we addressing localhost: port. this part I dislike - need to change it to load balancer in the future